src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -111,6 +111,10 @@ contract AllocationManagerUnitTests is EigenLayerUnitTestSetup, IAllocationManag
         // Set operatorSet to being valid
         avsDirectoryMock.setIsOperatorSetBatch(operatorSets, true);
 
+        // Set strategy to being valid for operator set
```

Comment:
Can we add three tests: 

1. Revert when you try to slash for a strategy that does not exist. The operator should have allocated to the strategy, but then the AVS removes the strategy from their list of strategies we should revert the slash
2. Revert when the operator tries to allocate for a strategy not added to the operatorSet
3. Validate that we can deallocate, even if the strategy is not part of the operator set

Reply:
I think we're still missing these?